tests/test_bgp_neighbor.rb

```diff
@@ -265,6 +265,26 @@ def test_dynamic_capability
     end
   end
 
+  def test_bfd
+    %w(default test_vrf).each do |vrf|
+      neighbor = create_neighbor(vrf)
+      if platform == :ios_xr
```

Comment:
Yup, there are probably a few more of those old platform checks floating about. We'll weed them out someday.